internal/events/handler.go

```diff
@@ -151,3 +156,27 @@ func (h *EventHandlerImpl) propagateDelete(e *DeleteEvent) {
 		panic(fmt.Errorf("unknown resource type %T", e.Type))
 	}
 }
+
+// setProgrammedStatus will set the Gateway and HTTPRoute conditions for when the Gateway is not programmed,
+// due to an error when attempting to reload nginx
+func setProgrammedStatus(objects []client.Object, statuses *state.Statuses) {
```

Comment:
`setProgrammedStatus` -> `setUnprogrammedStatus` ? (to match the comment for the function)